tests/libtest/.gitignore

```diff
@@ -1,4 +1,5 @@
 chkhostname
+lib1156
 lib15[0-9][0-9]
```

Comment:
How about instead make it even more generic and change `lib15[0-9][0-9]` to `lib1[0-9][0-9][0-9]` ?